packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx

```diff
@@ -74,6 +74,7 @@ describe('VitalsHeader: ', () => {
     expect(screen.getByText(/Last recorded/i)).toBeInTheDocument();
     expect(screen.getByText(/19 — May — 2021/i)).toBeInTheDocument();
     expect(screen.getByText(/Record vitals/i)).toBeInTheDocument();
+    expect(screen.getByRole('img', { name: /warning/i })).toBeInTheDocument();
```

Comment:
I'm curious what is role='img' here.. is that the icon?

Reply:
Yep, this one https://github.com/openmrs/openmrs-esm-patient-chart/blob/b98af1ecd254a97c37bd84d73ee4f6ed03565326/packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.component.tsx#L65/